spring-xd-integration-test/src/test/java/org/springframework/xd/integration/test/KafkaTests.java

```diff
@@ -35,10 +38,13 @@
 	@Test
 	public void kafkaSourceSinkTest() {
 		String data = UUID.randomUUID().toString();
-		String topic = UUID.randomUUID().toString();
+		String topicToUse = UUID.randomUUID().toString();
+		ZkClient zkClient = sources.getKafkaZkClient();
+		AdminUtils.createTopic(zkClient, topicToUse, 1, 1, new Properties());
+
 		String sinkStreamName = "ks" + UUID.randomUUID().toString();
-		KafkaSource source = sources.kafkaSource().topic(topic).ensureReady();
-		KafkaSink sink = sinks.kafkaSink().topic(topic).ensureReady();
+		KafkaSource source = sources.kafkaSource().topic(topicToUse).ensureReady();
```

Comment:
ensureReady can make sure that the topic is created.  There is duplicated code in KafkaSource and KafkaSink in the ensureReady method, this should be moved up to a new common abstract base class.